vorestation.dme

```diff
@@ -2230,12 +2230,6 @@
 #include "code\modules\mob\living\simple_animal\vore\solargrub_larva.dm"
 #include "code\modules\mob\living\simple_animal\vore\wolf.dm"
 #include "code\modules\mob\living\simple_animal\vore\zz_vore_overrides.dm"
-#include "code\modules\mob\living\simple_animal\vore\shadekin\_defines.dm"
-#include "code\modules\mob\living\simple_animal\vore\shadekin\ability_objects.dm"
-#include "code\modules\mob\living\simple_animal\vore\shadekin\ability_procs.dm"
-#include "code\modules\mob\living\simple_animal\vore\shadekin\shadekin.dm"
-#include "code\modules\mob\living\simple_animal\vore\shadekin\types.dm"
-#include "code\modules\mob\living\simple_animal\vore\shadekin\~defines.dm"

```

Comment:
all of this can be readded to the DME since the files are commented out

Reply:
re-added the removed lines.